src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+html,
+body,
+main,
+div,
+header,
+ul,
+li,
+a,
+h1,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  @include font($size: 10px, $weight: 400, $color: $main-accent-color);
+
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+button {
+  background-color: transparent;
+  color: inherit;
+  border-width: 0;
+  padding: 0;
+  cursor: pointer;
+}
+
+h1,
+p {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  @include flex(row, space-between, center);
+
+  max-width: 1200px;
+  margin: 0 auto;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    img {
+      @include set-size(40px, 40px);
+
+      display: block;
+    }
+  }
+
+  .nav {
+    &__list {
+      display: flex;
+    }
+    &__item:not(:last-child) {
+      margin-right: 20px;
+    }
+
+    &__link {
+      @include flex($direction: row, $items: center);
+
+      position: relative;
+      height: 60px;
+      color: $main-accent-color;
+      text-transform: uppercase;
+
+      &:hover {
+        color: $blue-accent-color;
+      }
+    }
+    .is-active {
+      color: $blue-accent-color;
+    }
+
+    .is-active::after {
+      @include set-size(100%, 4px);
```

Comment:
Ensure that the mixin `set-size` is defined in your mixins file. If it's not defined, this will cause an error during SCSS compilation.